source/_data/themes.yml

```diff
@@ -1,3 +1,13 @@
+- name: Mango
+  description: Simple & Light Theme
+  link: https://github.com/mango-tree/hexo-theme-mango
+  preview: https://mango-tree.github.io//
```

Comment:
Would you please fix preview url?
A trailing slash is a duplicate.

Reply:
Fixed it